pkg/features/kube_features.go

```diff
@@ -391,6 +393,12 @@ const (
 	// `externalTrafficPolicy: Cluster` services.
 	KubeProxyDrainingTerminatingNodes featuregate.Feature = "KubeProxyDrainingTerminatingNodes"
 
+	// owner: @bobbypage
+	// kep: https://kep.k8s.io/2000
+	//
+	// Enables support for graceful shutdown linux node.
+	LinuxGracefulNodeShutdown featuregate.Feature = "LinuxGracefulNodeShutdown"
```

Comment:
I'm happy with this route. Do we need to edit the KEP to match?

Reply:
@sftim I think it is need to update the KEP.

/hold 

Let's update the KEP first.